betelgeuse.py

```diff
@@ -644,9 +644,18 @@ def test_case(context, path, collect_only, project):
     '--parent-name',
     help='Name of parent Test Plan to link to.',
 )
+@click.option(
+    '--status',
+    help='Status of the Test Plan',
+    type=click.Choice([
+        'open',
+        'inprogress',
+        'done',
+    ])
+)
 @click.argument('project')
 @click.pass_context
-def test_plan(context, name, plan_type, parent_name, project):
+def test_plan(context, name, plan_type, parent_name, status, project):
```

Comment:
A couple of thoughts we can discuss:

* We could use the same approach as `test-run` and allow the user to pass `--custom-fields status=done` so that we can in theory also set other values such as start date, end date, etc.
* In theory, I should be able to create a new `Test Plan` and set the `status` to `inprogress` or `done`, so we should make sure that `status` is passed to the `create` method as well